indy_node/test/upgrade/helper.py

```diff
@@ -96,10 +96,20 @@ def __init__(self, backup_dir, backup_target, transform=lambda tool: None):
         transform(self.tool)
         self.p = multiprocessing.Process(target=self.tool.start)
         self.p.start()
+        logger.debug("NCTProcess was started with pid: {}". format(self.p.pid))
 
     def stop(self):
-        self.p.terminate()
+        logger.debug("Send stop to NCTProcess with pid: {}". format(self.p.pid))
         self.tool.server.close()
+        self.p.terminate()
+        # check that process with NetControlTool.start function really stop.
```

Comment:
Please correct: NetControlTool -> NodeControlTool.

Reply:
Agree. Let's do it in a separate PR.